generators/client/templates/angular/webpack/_webpack.prod.js

```diff
@@ -141,10 +120,10 @@ module.exports = webpackMerge(commonConfig({ env: ENV }), {
                 }
             }
         }),
-        new ngcWebpack.NgcWebpackPlugin({
-            disabled: false,
-            tsConfig: utils.root('tsconfig-aot.json'),
-            resourceOverride: ''
+        new AngularCompilerPlugin({
```

Comment:
The last time I tried this plugin it was too slow, did that improve if so we can remove the `NgcWebpackPlugin`

Reply:
It takes `30s` for the prod build on my local.

Reply:
did you try with or without entities? do a test with one of our Travis sample set